hack/gen-api-reference-docs.sh

```diff
@@ -26,14 +26,17 @@ REFDOCS_REPO="https://${REFDOCS_PKG}.git"
 REFDOCS_VER="v0.1.5"
 
 KNATIVE_SERVING_REPO="github.com/knative/serving"
+KNATIVE_SERVING_LOCAL_REPO="knative.dev/serving"
 KNATIVE_SERVING_COMMIT="${KNATIVE_SERVING_COMMIT:?specify the \$KNATIVE_SERVING_COMMIT variable}"
 KNATIVE_SERVING_OUT_FILE="serving.md"
 
 KNATIVE_EVENTING_REPO="github.com/knative/eventing"
+KNATIVE_EVENTING_LOCAL_REPO="knative.dev/eventing"
```

Comment:
let's rename `_LOCAL_REPO` to `_IMPORT_PATH` if you don't mind.

Reply:
@ahmetb done, PTAL